vuepress/docs/docs/client.md

```diff
@@ -69,6 +69,20 @@ In order to supply more configuration options to the code generator, you can use
       )
 ```
 
+The path where the generator will look for schemas can be customized by overriding the `calibanSources` settings:
+
+```scala
+Compile / caliban / calibanSources := file("caliban")
+```
+
+If you want to only include certain files in a directory, you can override that as well with an explicit `caliban / sources` entry:
```

Comment:
This isn't actually true, since `calibanSources` is a `File`, not a `List[File]` -- the comment doesn't scale and will trip up users

Reply:
Updated, hopefully it's correct now